stable_audio_tools/interface/gradio.py

```diff
@@ -665,7 +667,21 @@ def create_ui(model_config_path=None, ckpt_path=None, pretrained_name=None, pret
     else:
         model_config = None
 
-    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    try:
+        has_mps = platform.system() == "Darwin" and torch.backends.mps.is_available()
+    except Exception:
+        # In case this version of Torch doesn't even have `torch.backends.mps`...
+        has_mps = False
+
+    if has_mps:
+        device = torch.device("mps")
+    elif torch.cuda.is_available():
+        device = torch.device("cuda")
+    else:
+        device = torch.device("cpu")
+
+    print("Using device:", device)
```

Comment:
Remove this print and this is probably good to go

Reply:
I think it's very useful to see at a glance which device is being used. For instance, seeing `Using device: cpu` when you're on an ostensibly CUDA machine...